includes/log-handlers/class-wc-log-handler-file.php

```diff
@@ -266,7 +266,7 @@ public function remove( $handle ) {
 	protected function should_rotate( $handle ) {
 		$file = self::get_log_file_path( $handle );
 		if ( $file ) {
-			if ( $this->is_open( $handle ) ) {
+			if ( $this->is_open( $handle ) && is_resource( $this->handles[ $handle ] ) ) {
```

Comment:
@sirreal I just copied your own code. There is other example of `$this->is_open( $handle ) && is_resource( $this->handles[ $handle ] )` in the close method.
And if `is_open` is able to prune such a thing he should not seen any error already: https://github.com/woocommerce/woocommerce/issues/14544

So should I change the `is_open` method?